confgenerator/logging_processors.go

```diff
@@ -523,3 +523,28 @@ func (p LoggingProcessorExcludeLogs) Processors(ctx context.Context) ([]otel.Com
 func init() {
 	LoggingProcessorTypes.RegisterType(func() LoggingProcessor { return &LoggingProcessorExcludeLogs{} })
 }
+
+// LoggingProcessorMacro implements the methods required to describe a pipeline with one or more log processors.
+type LoggingProcessorMacro interface {
+	Type() string
+	// Processors returns slice of logging processors. This is an intermediate representation before sub-agent specific configurations.
+	InternalLoggingProcessors(ctx context.Context) []InternalLoggingProcessor
```

Comment:
Rename `InternalLoggingProcessor` -> `Expand`. Following the idea of the expanding this macro.